pcsx2/Patch_Memory.cpp

```diff
@@ -420,6 +423,25 @@ void _ApplyPatch(IniPatch *p)
 			handle_extended_t(p);
 			break;
 
+		case SHORT_LE_T:
+			ledata = SwapEndian(p->data, 16);
+			if (memRead16(p->addr) != (u16)ledata)
+				memWrite16(p->addr, (u16)ledata);
+			break;
+
+		case WORD_LE_T:
+			ledata = SwapEndian(p->data, 32);
+			if (memRead32(p->addr) != (u32)ledata)
+				memWrite32(p->addr, (u32) ledata );
```

Comment:
remove the spacing on the ledata parameter please :)  so it should be memWrite32(p->addr, (u32)ledata);

Also do the same for the memWrite64 below.  (last thing, I promise lol)